shell-pager-ess.el

```diff
@@ -0,0 +1,191 @@
+;;; shell-pager-ess.el --- Support for ESS  -*- lexical-binding: t -*-
+
+
+;;; Commentary:
```

Comment:
Heads up... I'm eventually going to add this to the files. You can add it now or please just acknoledge and I will do later.

;; This package is free software; you can redistribute it and/or modify
;; it under the terms of the GNU General Public License as published by
;; the Free Software Foundation; either version 3, or (at your option)
;; any later version.

;; This package is distributed in the hope that it will be useful,
;; but WITHOUT ANY WARRANTY; without even the implied warranty of
;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
;; GNU General Public License for more details.

;; You should have received a copy of the GNU General Public License
;; along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.

Reply:
Added to the new commit.